lib/veritech-server/src/handlers.rs

```diff
@@ -154,6 +168,7 @@ pub async fn process_request(
             return Err(HandlerError::InvalidIncomingSubject(subject));
         }
     }
+    info!(%execution_kind, %execution_id, "finished execution");
```

Comment:
Now we log post dispatch

Reply:
Long term I'm not sure we want too `info` level logs in these handlers (as above). It becomes way too noisy given how often these fire and I'm sensing these are here as a lack of trust on the trace spans (I know these lines have been here for a while, but today these are striking me as potential excessive noise as soon as we have our bug tracked down).

Reply:
Removed since the `handlers_doing_work` metric and the log above the dispatch tell the right story